packages/shared/src/utils/constants/contract.ts

```diff
@@ -602,8 +607,14 @@ export const getContractConfig = (is_high_low?: boolean) => ({
 // TODO we can combine getContractTypeDisplay and getContractTypePosition functions.
 the difference between these two functions is just the property they return. (name/position)
 */
-export const getContractTypeDisplay = (type: string, is_high_low = false, show_button_name = false) => {
+export const getContractTypeDisplay = (
+    type: string,
+    is_high_low = false,
+    show_button_name = false,
+    show_main_title = false
+) => {
     const contract_config = getContractConfig(is_high_low)[type as TGetSupportedContracts] as TContractConfig;
+    if (show_main_title) return contract_config?.main_title ?? '';
     return (show_button_name && contract_config?.button_name) || contract_config?.name || '';
 };
```

Comment:
Can this be made into accepting an object instead of these optional props? Currently, the function looks quite unusable to call. 

Something like this: 
```
export const getContractTypeDisplay = (
    type: string,
    options: ContractOptions = {}
) => {
    const { isHighLow = false, showButtonName = false, showMainTitle = false } = options;

    const contractConfig = getContractConfig(isHighLow)[type as TGetSupportedContracts] as TContractConfig;

    if (showMainTitle) {
        return contractConfig?.main_title ?? '';
    }

    return (showButtonName && contractConfig?.button_name) || contractConfig?.name || '';
};
```

Reply:
Sure, I can try) But I need to change all places where this function is called, will work in this

Reply:
What I mean, just pass it in those places were you need these booleans, it should still have default values, so it shouldn't affect existing logic where we don't pass anything. The reason to do this is that if let's say we need to pass only `showMainTitle`, right now we'd need to pass 2 falses ahead of it to let it know what we refer too.

Reply:
Yep, it worked, I'll also refactor getTradeTypeName function as it works similar